src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -25,17 +26,39 @@
     description = "Все методы для работы с пользователями",
 )
 
-class AuthController(private val userService: UserService) {
+class AuthController(
+    private val userService: UserService,
+    val ftService: FTService
+) {
+
+    private val userCanCreate: String = "userCanCreate"
+    private val userCanGet: String = "userCanGet"
+    private val userCanDelete: String = "userCanDelete"
+    private val userCanLogin: String = "userCanLogin"
+    private val userCanUpdate: String = "userCanUpdate"
+    private val userCanlogout: String = "logout"
 
     @PutMapping("/user/{id}")
     @Operation(summary = "Обновление данных пользователя")
-    fun updateUser(@PathVariable id: Int, @RequestBody user: UserDTO) {
+    fun updateUser(@PathVariable id: Int, @RequestBody user: UserDTO): Any {
```

Comment:
🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline